src/NuGetPackageVerifier/Rules/AssemblyHasAttributeHelper.cs

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using Mono.Cecil;
+
+namespace NuGetPackageVerifier.Rules
+{
+    public class AssemblyHasAttributeHelper
+    {
+        public static void GetAssemblyAttributesData(PackageAnalysisContext context)
+        {
+            if (context.AssemblyData.Count == 0)
```

Comment:
You can earn up to 5 Pranav Points if you use `!x.Any()` instead of `x.Count == 0`.